From def409e44cf0e0e4fe01c0a535610e70aab4fde5 Mon Sep 17 00:00:00 2001
From: Mike Wiederhold <mikewied06@gmail.com>
Date: Fri, 22 Jul 2016 14:33:09 -0700
Subject: [PATCH] MB-19885: Rename json variable to prevent name clash

There is a json module that we are loading so we shouldn't have
json named variables.

Change-Id: I6434472517f36932928c0e4462034b31d71cdeca
Reviewed-on: http://review.couchbase.org/66111
Well-Formed: buildbot <build@couchbase.com>
Reviewed-by: Eben Haber <eben@couchbase.com>
Tested-by: Michael Wiederhold <mike@couchbase.com>
---

diff --git a/info.py b/info.py
index db940f2..7230de4 100644
--- a/info.py
+++ b/info.py
@@ -39,15 +39,15 @@
         data = rest.restCmd('GET', '/nodes/self',
                             user, password, opts)
 
-        json = rest.getJson(data)
+        result = rest.getJson(data)
 
         for x in ['license', 'licenseValid', 'licenseValidUntil']:
-            if x in json:
-                del(json[x])
+            if x in result:
+                del(result[x])
 
         if cmd == 'server-eshell':
-            node = json['otpNode']
-            cookie = json['otpCookie']
+            node = result['otpNode']
+            cookie = result['otpCookie']
 
             if vm != 'ns_server':
                 rest = util.restclient_factory(server, port, {'debug':self.debug}, ssl)
@@ -67,9 +67,9 @@
             p = subprocess.call([self.getErlPath(), '-name', name, '-setcookie',
                                  cookie, '-hidden', '-remsh', node])
         elif cmd == 'get-server-info':
-            return json
+            return result
         else:
-            print json.dumps(json, sort_keys=True, indent=2)
+            print json.dumps(result, sort_keys=True, indent=2)
 
     def getErlPath(self):
         bin = os.path.join(os.path.dirname(os.path.abspath(sys.argv[0])), '..', '..', 'bin')